dev/standalone/RUN_INTEGRATION.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
Could/should this be a separate pr?

Reply:
dev/common/RUN_INTEGRATION.sh is going away and this continues the trend of having a script per compose profile.